app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt

```diff
@@ -183,15 +183,6 @@ class AddProfileActivityTest {
     assertThat(label).isEqualTo(context.getString(R.string.add_profile_activity_label))
   }
 
-  @Test
-  fun testAddProfileActivity_hasRequiredTextPrefixedWithStar() {
```

Comment:
Instead, I think you'd want to add tests to ensure all the fields that are required are, indeed, marked as required (per their prefix helper text).

Reply:
@BenHenning PTAL
Tests to ensure " all the fields that are required are, indeed, marked as required (per their prefix helper text) " have been added with the latest push. 
I used this  [document, line 784](https://github.com/material-components/material-components-android/blob/master/tests/javatests/com/google/android/material/textfield/TextInputLayoutIconsTest.java) to guide me on how to write the tests. I'm looking forward to your feedback regarding my approach.

Reply:
@BenHenning PTAL
The verification process has been established and removing the text helper does result in the test failing, as shown by the test fail images below.
![confirmpinerror](https://user-images.githubusercontent.com/31986629/197153352-0fa5338f-15bb-4225-8f2d-ec4f6b15e668.PNG)
![nameerror](https://user-images.githubusercontent.com/31986629/197153360-bc19c220-b4d2-4e9f-8d5d-4608ddcff2af.PNG)
![pinerror](https://user-images.githubusercontent.com/31986629/197153366-6be79a04-e594-4b6c-89bb-12bfb1e6bfe9.PNG)

Reply:
Ah, I wonder if it's fixed because of the new action that was introduced for helper text. Either way, thanks for verifying that they're failing as expected.